harvard-lts/epadd-curator-app
# Zipping fix

**Ensures that the directories are being zipped properly**
* * *

**Ticket Link**: https://jira.huit.harvard.edu/browse/LTSEPADD-38

# What does this Pull Request do?
Zips the exported directory into a 7z file

# How should this be tested?
Unit tests:
Follow the testing instructions in the readme: https://github.com/harvard-lts/epadd-curator-app/blob/LTSEPADD-38/README.md#running-tests

Manual somewhat integration test
1.  Empty epadd-export-dev bucker with `aws s3 rm --recursive s3://epadd-export-dev/test/ --profile epadd-export-dev-rw `  

2. Call http://ltsds-cloud-dev-1.lib.harvard.edu:10586/testExport/{aUniqueOSN}

3. Confirm that the batch directory is makes it into the dropbox with a 7z zip under it.  Currently the batches are ending with a '?' which is being worked on.  Dropbox is `/drs2dev/drsfs/dropbox/epadddev_secure/incoming` on the docker dev cloud servers.


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? no
